packages/tailwindcss-intellisense/src/lsp/server.ts

```diff
@@ -156,6 +156,8 @@ connection.onInitialize(
             '.',
             // config/theme helper
             '[',
+            // class name including '-'
+            '-',
             // TODO: restart server if separater changes?
```

Comment:
```suggestion
            // TODO: restart server if separator changes?
```
since you already touch the preceding line, I believe it's a good opportunity to fix the typo :wink:

Reply:
sure...but the whole file seems gone after I did a rebase, updated PR in the new file 🙂️.

Reply:
Alright then nevermind